package.json

```diff
@@ -71,19 +72,19 @@
     "react": "16.9.0",
     "react-dom": "16.9.0",
     "react-native": "^0.61.5",
-    "react-native-shared-element": "^0.5.5",
-    "react-navigation-stack": "^2",
+    "react-native-shared-element": "^0.5.6",
     "react-test-renderer": "16.9.0",
     "release-it": "^12.4.3",
     "scheduler": "^0.18.0",
     "typescript": "^3.7.4"
   },
   "peerDependencies": {
-    "@react-navigation/core": "^3.0.0",
+    "@react-navigation/core": "^5.2.0",
```

Comment:
core shouldn't be a peer dep here

Reply:
I had some issues with getting proper TypeScript-definitions when starting on this branch without adding `core`. But I can't seem to reproduce it now, so probably PEBKAC. 🤷‍♂

Reply:
I've removed the peer dep, thanks @satya164 👍